### PR TITLE
Adding bambuser broadcaster to the directory

### PR DESCRIPTION
**Pull request by fansilva1991**
# 📝 Why & how

Adding the [react-native-bambuser-broadcaster](https://github.com/bambuser/react-native-bambuser-broadcaster) to the directory.

# ✅ Checklist

- [x] Added library to **`react-native-libraries.json`**
- [ ] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

**Comment by fansilva1991** [comment]
Hi @Simek, this library is still compatible with the latest version of React Native; 

I'm using it with version ```0.71.3```

Sadly, there is no alternative for this library since this has not been updated by the vendor ( Bambuser );

I think we should add this since this is the only alternative available, and developers can even use it to sign to their teams that the library is unmaintained, making them sure about the risks of using it.

But I will understand if we decide not to add this to the directory :)

Thanks for the response

File: react-native-libraries.json [hunk 1]
```diff
@@ -9518,5 +9518,19 @@
...
+    "android": true,
+    "web": false,
+    "expo": false,
+    "windows": false,
+    "macos": false,
+    "tvos": false,
+    "unmaintained": true,
+    "dev": false,
+    "template": false,
+    "newArchitecture": false
```
Comment (Simek) [inline react-native-libraries.json:9534]: Thank you for the explanation and checks! 👍  If the library works with the latest version of React Native it is fine to add it, besides the `unmaintained` flag I think, so LGTM.

Let's just clean up the entry, before we merge that in:
```suggestion
    "android": true,
    "unmaintained": true
```

Comment (fansilva1991) [reply react-native-libraries.json]: It is done as suggested :)

**Review by Simek — APPROVED** [review]
Thank you for the updates! 🚀

**Pull request merged**